homeassistant/components/huawei_lte/strings.json

```diff
@@ -27,6 +29,7 @@
       "user": {
         "data": {
           "password": "[%key:common::config_flow::data::password%]",
+          "unverified_https": "Use unverified HTTPS",
```

Comment:
Maybe we should use `[%key:common::config_flow::data::verify_ssl%]` here instead and adjust referring strings accordingly.

"Verify SSL certificate" is somewhat unfortunate phrasing though, in practice it's TLS we do nowadays, but maybe the general public is better accustomed to SSL than that. Anyway HTTPS would fit better as a term here, so in that sense we might want to keep this phrasing. Thoghts welcome.

Reply:
The existing language is used by 20 integrations, so it probably makes sense to use the same here?
Could the existing phrase be adjusted to something like "Verify SSL/TLS certificate"?

Reply:
Yeah, I'll follow suit here, 9e2060ae8cbee05e0ad3be3c64e4372231e93e26 switches to the existing config and string keys.

We could see what others think about adjusting the existing phrase, but I think it's off topic for this PR. "Verify TLS/SSL certificate, "Verify TLS certificate", "Verify SSL/TLS certificate", "Verify certificate" could be candidates worth considering. ("Verify HTTPS certificate" would be nice, but the key might be already in use for other TLS verifications besides HTTPS.)

Reply:
BTW I would prefer if we'd merge #85929 before this instead of this to its branch in order to not unnecessarily balloon it. Kind of would make sense to mark this as draft for that reason, but it's really not a draft but all ready for review, so I'd rather not.

Reply:
#85929 has been merged, this one rebased and good to go.